frontend/client/components/Proposals/Filters/index.tsx

```diff
@@ -32,18 +40,17 @@ export default class ProposalFilters extends React.Component<Props> {
         <div style={{ marginBottom: '1rem' }} />
 
         <Card title="Filter" extra={<a onClick={this.resetFilters}>Reset</a>}>
-          <h3>Proposal stage</h3>
```

Comment:
Note the header here is being removed. I was going to rename it since we're no longer just filtering on stage types, but the UI felt cleaner doing away with it altogether. Let me know if we should add it back in.